environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -108,6 +108,8 @@
     public static final String PROXYCONFIG_NAME = "Name of the proxyconfig of the environment.";
     public static final String PROXYCONFIG_RESPONSE = "ProxyConfig attached to the environment.";
 
+    public static final String CLOUDBREAK_VERSION = "The version of the Cloudbreak build used to create the environment.";
```

Comment:
I think we should highlight Environment service here. I know this is overall one repo and same version for each service but we should clearly separate services